lib/Common/DataStructures/BigInt.h

```diff
@@ -50,8 +51,8 @@ namespace Js
         void Subtract(BigInt *pbi);
         int DivRem(BigInt *pbi);
 
-        int32 Clu(void);
-        uint32 Lu(int32 ilu);
```

Comment:
How much existing code uses this data structure? Is it possible that we could rename these methods to be more reasonable without changing a ton of unrelated code to match?

Reply:
Pretty much the code here is used to convert a string to double. I found about 30-40 references to this code.

---
In reply to: [214254468](https://github.com/Microsoft/ChakraCore/pull/5657#discussion_r214254468) [](ancestors = 214254468)